index.js

```diff
@@ -240,5 +240,13 @@ client.on('guildMemberAdd', async member => {
     }
   }
 });
+client.on('commandError', (command, err) => {
+  var commandErrorEmbed = new MesageEmbed()
+  .setColor('RANDOM')
+  .setTitle('Error')
+  .addField('Command that is being executed',command)
```

Comment:
Should be `command.name` or `command.memberName` to return the commands name instead of the Commands Register Array for the errored command